src/main/java/com/minecolonies/coremod/util/LanguageHandler.java

```diff
@@ -26,58 +26,15 @@ private LanguageHandler()
     /**
      * Send a message to the player.
      *
-     * @param player the player to send to.
-     * @param key the key of the message.
+     * @param player  the player to send to.
+     * @param key     the key of the message.
      * @param message the message to send.
      */
     public static void sendPlayerMessage(@NotNull final EntityPlayer player, final String key, final Object... message)
     {
         player.sendMessage(buildChatComponent(key, message));
     }
 
-    /**
-     * Localize a string and use String.format().
-     *
-     * @param key  translation key.
-     * @param args Objects for String.format().
-     * @return Localized string.
-     */
-    public static String format(final String key, final Object... args)
-    {
-        final String result;
-        if(args.length == 0)
-        {
-            result = new TextComponentTranslation(key).getUnformattedText();
-        }
-        else
-        {
-            result = new TextComponentTranslation(key, args).getUnformattedText();
-        }
-        return result.isEmpty() ? key : result;
-    }
-
-    /**
-     * Send message to a list of players.
-     *
-     * @param players the list of players.
-     * @param key key of the message.
-     * @param message the message.
-     */
-    public static void sendPlayersMessage(@Nullable final List<EntityPlayer> players, final String key, final Object... message)
-    {
-        if (players == null || players.isEmpty())
-        {
-            return;
-        }
-
-        ITextComponent textComponent = buildChatComponent(key, message);
-
-        for (@NotNull final EntityPlayer player : players)
-        {
-            player.sendMessage(textComponent);
-        }
-    }
-
     private static ITextComponent buildChatComponent(final String key, final Object... message)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 28 to the 20 allowed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)